Automattic/wp-calypso
# Domain picker: Render unavailable domains when the user searches for a FQDN

#### Changes proposed in this Pull Request

* This PR is the first step to fix [this](https://github.com/Automattic/wp-calypso/issues/41219) issue. 
* This preliminary first step only adds the FQDN to the results with unavailable status.
* The next step should be explaining why the domain isn't available by querying the availability endpoint. This one is a bit of a rabbit hole as we'll have to package-ize [these.](https://github.com/Automattic/wp-calypso/blob/trunk/client/lib/domains/registration/availability-messages.js#L40-L390) 

#### Testing instructions

1. go to [/new](https://calypso.live/new?branch=update/show-fqdn-queries).
2. reach the domain step.
3. search for an unavailable FQDN (eg google.com), the first item should be your query but marked unavailable.
4. search for an available FQDN (eg hello-testing-domain.com). You should see a result with your domain that is recommended.
5. search for an unavailable .blog FQDN (tim.blog), the first item should be your query but marked unavailable.

Fixes the first OC from: https://github.com/Automattic/wp-calypso/issues/41219
Fixes #41066
